### DISCUSSION TITLE
Some suggestions after testing

### DISCUSSION THREAD

**Discussion by Tagael** [PeasantFoundry/Mechtrauma#89] [Ideas]
I was able to test the mod Mechtrauma unstable extensively and built a whole submarine with all systems for it. I separate my feedback into Bugs (Issues page) and Suggestions as requested

1.Secure Lockers
there is no secure locker (safes or diving suite locker). Not a bug, more a missing important feature. 

2.Oxygen Compressors: 
I would still charge at least 2 or rather 3 O2 bottles in a compressor. You already have to deal with a much higher difficulty that the suits in the lockers are no longer filled automatically, that you need Co2 filters and electrical motors for the compressors and also that the compressors extract the ambient oxygen (I would make this feature as optional adjustable, it was very annoying). In addition, the compressors have also become twice as big. As a result, they have only a third of the capacity with twice the size. For a longer diving trip you need 5 or 6 tanks and for that you would have to take all oxygen tanks on board, if you have so many compressors on board at all. With all the disadvantages to be able to load only one bottle is too much in my opinion. 
3.Detoriation Times: 
I would set the default detoriation times of the items significantly high. I understand the intention behind the short times but the problem is: If a mechanic or engineer can't get off the engine/turbine/boiler etc. at all because they are almost only down there, then many other things on the submarine can't be done (damage management, diving missions, crafting etc). Man br those also fall into their area of responsibility. Things are supposed to break down and be maintained, that's great, but in significantly more time. Also, I would set the maximum possible detoriation times in the mod menu still significantly upwards, so that you simply have more setting freedom here.

In Neurotrauma, the bots can no longer fill the role of Medical Officer and in Mechtrauma, the bots can no longer fill the role of Engineer and Mechanic. I would therefore also pay attention to the playability of the mod. As said, the problem can be solved by simply raising the standard detoriation times and increasing the maximum detoriation times in the menu. 

4.Information about the mod:
 It took me a few days to read up on the mod, as the information about the mod was currently scattered across many pages and there was some important information everywhere. There are currently 5 places to read to get information: 
1. Mechtrauma Alpha Steam Page 
2. Mechtraume Unstable Steam Page 
3.Mechtrauma Wiki 
4. Mechtrauma Release Notes 
5. Legacy Documentation 
I am aware that the mod is in progress and the information is currently not complete and that is not the problem. I would like to point out that they are in 5 places and in different forms. I would focus on one spot and only update it there. A short description of what's new is enough on the Steampage, but without an explanation of how something is wired or how it works in detail. For the average gamer (not a programmer) there should be one central site where all the information can be found.I would do all of that on the Mechtrauma Wiki. 
I hope my comments will help to further improve this great mod!

**Comment by MapleWheels** [comment]
Thank you for your feedback. I think it's time to bring this mod development out of hiatus lol.

**Reply by PeasantFoundry** [reply]
Thanks for the feedback! Your suffering matters to us!

1.Secure Lockers
there is no secure locker (safes or diving suite locker). Not a bug, more a missing important feature.

A: There is a new security control system based on ID cards and/or passwords that has been prototyped and would give the sub designer the ability to handle this.

Basically, the security control can be linked to items such as doors and containers. In the case of a container (cabinet/locker) the container couldn't be interacted with (picked/opened) unless the security control enabled it. So yes, you could have gear lockers that require a password or id card. 

2.Oxygen Compressors:
A: I'm so glad this feature annoyed you! However, I will point out that it is optional as you can just use the vanilla compressors if you want. Also the oxygen tank station will fill an 02 bottle in a matter of seconds.
Having said that... There probably should be an oxygen refill that can have more bottles inside of it at once. And I'll probably do that whenever I circle back around to it.

3.Detoriation Times:
A: Which items do you want to deteriorate more slowly? You can already stop oil from deteriorating and make circulator pumps, bearings, and filters last 60 minutes. I think the electric motor and c02 filter deterioration rates still need to be added to the settings. 

We've tried to make the mod more forgiving towards bots where we can but it's really intended to be run with humans. Bots still make fine welders and gunners. 

4.Information about the mod:
A: Yeap, that's the plan!
